src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -418,6 +426,9 @@ export class ChallengesubmitComponent implements OnInit {
     FORM_DATA.append('method_description', self.globalService.formValueForLabel(self.components, 'method_description'));
     FORM_DATA.append('project_url', self.globalService.formValueForLabel(self.components, 'project_url'));
     FORM_DATA.append('publication_url', self.globalService.formValueForLabel(self.components, 'publication_url'));
+    if (self.validFileUrl === true && self.urlboxchecked) {
```

Comment:
@Suryansh5545 Also there is an error of key error of `file_url` when submitting using URL. So you need to look if these condition working perfectly for both the cases.